.github/workflows/ci.yml

```diff
@@ -44,9 +44,9 @@ jobs:
         features:
           - # default
           - --no-default-features
-          - --features track-caller
+          - --no-default-features --features track-caller
+          - --no-default-features --features auto-install
```

Comment:
How does changing to "--no-default-features" for eyre's track-caller and auto-install tests relate to adding color-eyre to the workspace?

Reply:
Not sure, I think it was Pavan who added that to increase the test coverage for feature combinations. This is one of the earliest commits

Reply:
Well, it's replacing two feature combinations, i'm not sure that it expands the test coverage. If we want to increase the test coverage we could keep both `--features` and `--no-features`.